preferencesfx/src/main/java/com/dlsc/preferencesfx/view/PreferencesFxDialog.java

```diff
@@ -87,10 +87,7 @@ private void setupDialogClose() {
       if (persistWindowState) {
         saveWindowState();
       }
-      if (saveSettings) {
-        model.saveSettingValues();
-        model.fireEvent(PreferencesFxEvent.preferencesSavedEvent());
-      }
+     model.saveSettings();
```

Comment:
Good! I prefer the way of calling it here instead of defining it as we did previously.